Mod2/Labs/Week1/SQLCRUD.md

```diff
@@ -0,0 +1,50 @@
+## Lab Objectives
+* Practice using `SELECT` `UPDATE` `REMOVE` and `INSERT`
+
+** Instructor Note ** Students are going to build upon their work from yesterday's lab.  It would be helpful if they are in the same groups, if possible.  If students need to start from a fresh database setup, you can provide them with this script (it should work if their database is called 'Library').
+```SQL
+DROP TABLE books, patrons;
+-- this should be a list of their tables.
+
+CREATE TABLE patrons (
+	id serial PRIMARY KEY,
+	FirstName varchar(50),
+	LastName varchar(50),
+	PhoneNumber varchar(10)
+);
+
+CREATE TABLE books (
+	id serial PRIMARY KEY,
+	Title varchar(255),
+	Author varchar(255),
+	CheckedOutBY int REFERENCES patrons
+);
+```
+
+
+## Practice
+
+In small groups, use PgAdmin to create, update, remove, and query some records for the Library database we created yesterday.  There are some required activities below, but feel free to experiment with your database!  If you ever feel that you need a fresh start for your database, reach out to your instructors.
+
+### Add Books to the Library and Create Library Patrons
+
+1. Use `INSERT` to create at least 5 patrons for your library (these can be real, or fake people).
+2. Use `INSERT` to create at least 10 books in your library.  The books can be all unique, or you could have multiple copies of more popular titles!  (books can also be real, or fake)
```

Comment:
Maybe add a minimum "uniqueness" to this step as well. I originally wrote a lot in this comments, but the patrons should all have different first names, last names, and phone numbers (for the sake of simplicity). For the books, maybe at least 3 different titles and 3 different authors. Other than that, they can duplicate as necessary.